### PR TITLE
📝(profile) enrich documentation

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

The documentation had to be expanded as some descriptions were quite laconic for
the examination for the first release.

## Proposal 

Enrich README and templates descriptions.

### CODE REVIEW

File: statements/uploaded_video.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## Description
 
-This event is emitted when a user has uploaded a file of `video` type.
-NB: this template is inspired from templates from [Video Profile](https://profiles.adlnet.gov/profile/90b2c849-d744-4d0c-8bd0-403e7859a35b).
+A user has uploaded a file of `audvideoio` type to an LMS or an e-learning platform.
```
Comment (wilbrdt) [inline statements/uploaded_video.md:5]:
```suggestion
A user has uploaded a file of `video` type to an LMS or an e-learning platform.
```
:grin:

File: statements/accessed_page.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-The actor has accessed a page of a LMS or a website.
+A user has accessed or navigated a file on a LMS or an e-learning platform.
```
Comment (wilbrdt) [inline statements/accessed_page.md:5]:
```suggestion
A user has accessed or navigated a page on a LMS or an e-learning platform.
```

**Review by wilbrdt — APPROVED** [review]
Some small remarks but LGTM! :tada:

File: statements/uploaded_video.md (hunk 1 above)
Comment (quitterie-lcs) [reply statements/uploaded_video.md:5]: Woopsie

**Pull request merged**